src/System.Diagnostics.Process/tests/ProcessStreamReadTests.netcoreapp2.1.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
```

Comment:
I think I mentioned to call it netcoreapp2.1 but we are trying to go with the versionless convention. Could you please rename this two files to .netcoreapp.cs versionless?